packages/liveblocks-node/src/client.ts

```diff
@@ -1,3 +1,4 @@
+import type { CommentData, ThreadData } from "@liveblocks/core";
```

Comment:
```suggestion
/**
 * NOTE: only types should be imported from @liveblocks/core.
 * This is because this package is made to be used in Node.js, and
 * @liveblocks/core has browser-specific code.
 */
import type { CommentData, ThreadData } from "@liveblocks/core";
```